generated/_task_modules/channels/ai_channel.py

```diff
@@ -2618,11 +2657,13 @@ def ai_current_units(self):
             self._handle, self._name)
         check_for_error(error_code)
 
+
+
     @property
-    def ai_custom_scale(self):
```

Comment:
It looks like there was a special case for this property since we aren't setting the name in python, we are setting an instance of a class (but we only need the name). Definitely need to add a special case for this in the generator somehow.